stdlib/builtins.pyi

```diff
@@ -1066,35 +1066,35 @@ def getattr(__o: object, __name: str, __default: None) -> Any | None: ...
 def getattr(__o: object, __name: str, __default: bool) -> Any | bool: ...
 @overload
 def getattr(__o: object, __name: str, __default: _T) -> Any | _T: ...
-def globals() -> dict[str, Any]: ...
+def globals() -> dict[str, object]: ...
 def hasattr(__obj: object, __name: str) -> bool: ...
 def hash(__obj: object) -> int: ...
 def help(*args: Any, **kwds: Any) -> None: ...
 def hex(__number: int | SupportsIndex) -> str: ...
 def id(__obj: object) -> int: ...
-def input(__prompt: Any = ...) -> str: ...
+def input(__prompt: object = ...) -> str: ...
 @overload
 def iter(__iterable: Iterable[_T]) -> Iterator[_T]: ...
 @overload
 def iter(__function: Callable[[], _T | None], __sentinel: None) -> Iterator[_T]: ...
 @overload
-def iter(__function: Callable[[], _T], __sentinel: Any) -> Iterator[_T]: ...
+def iter(__function: Callable[[], _T], __sentinel: object) -> Iterator[_T]: ...
 
 if sys.version_info >= (3, 10):
     def isinstance(
-        __obj: object, __class_or_tuple: type | types.UnionType | Tuple[type | types.UnionType | Tuple[Any, ...], ...]
+        __obj: object, __class_or_tuple: type | types.UnionType | Tuple[type | types.UnionType | Tuple[object, ...], ...]
```

Comment:
Any feels more appropriate here because we're using it due to an insufficient expressive type system (we need recursive types to express the real type).

Reply:
I see; yes, that makes sense